src/app/ubs/ubs-admin/components/ubs-admin-customers/ubs-admin-customers.component.html

```diff
@@ -343,18 +343,28 @@
         </div>
       </mat-header-cell>
       <mat-cell *matCellDef="let row">
-        <div
-          class="column_cell"
-          [matTooltip]="row[column.title.key]"
-          matTooltipShowDelay="700"
-          matTooltipClass="tooltip_style"
-          [class.pointer]="
-            column.title.key === 'clientName' || column.title.key === 'number_of_orders' || column.title.key === 'violations'
-          "
-          (click)="openPages(column.title.key, row)"
-        >
-          {{ row[column.title.key] }}
+        <div *ngIf="column.title.key === 'chatLink'; else readonly">
+          <button class="action" (click)="openPopUp(column, row[column.title.key], row.userId)">
+            {{ row[column.title.key] ? '&#9998;' : '&#43;' }}
+          </button>
+          <button *ngIf="row[column.title.key]" class="action" (click)="onOpenChat(row[column.title.key])">
+            {{ 'ubs-tables.go-to-chat' | translate }}
+          </button>
         </div>
+        <ng-template #readonly>
+          <div
+            class="column_cell"
+            [matTooltip]="row[column.title.key]"
+            matTooltipShowDelay="700"
+            matTooltipClass="tooltip_style"
+            [class.pointer]="
+              column.title.key === 'clientName' || column.title.key === 'number_of_orders' || column.title.key === 'violations'
```

Comment:
The logic can be in the component, making the template cleaner and easier to manage:
```
private readonly pointerColumns: string[] = ['clientName', 'number_of_orders', 'violations'];

get isPointerColumn(): boolean {
  return this.pointerColumns.includes(this.column.title.key);
}
```